test/e2e/framework/pods.go

```diff
@@ -0,0 +1,32 @@
+package framework
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (f *Framework) FindPodsByAppLabel(cluster ClusterIndex, appName string, namespace string, expectedCount int) *v1.PodList {
```

Comment:
may be AwaitPodsByAppLabel? 

(since we wait for them, and not only find?)

Reply:
yeah

Reply:
I think Find is more appropriate. we are waiting till all pods are found which is the default behavior. We have to wait for output in every case.

Reply:
I'm find either way - ok to leave it as Find?

Reply:
I think Await explains better what the function will do (I would expect Find to only return the existing pods "now" and just exit).

Reply:
Oh, comment does not appear here.

In this case I think it's better Await, because otherwise it's confusing, I'd expect it to output the pods right now, even if they are not present.